mmv1/third_party/terraform/services/bigtable/resource_bigtable_table.go

```diff
@@ -103,6 +103,29 @@ func ResourceBigtableTable() *schema.Resource {
 				ValidateFunc: verify.ValidateDuration(),
 				Description:  `Duration to retain change stream data for the table. Set to 0 to disable. Must be between 1 and 7 days.`,
 			},
+
+			"automated_backup_policy": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"retention_period": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							ValidateFunc: verify.ValidateDuration(),
+							Description:  `How long the automated backups should be retained. The only supported value at this time is 3 days.`,
```

Comment:
Shall we also mention that "set to 0 to disable"? Same thing for `frequency`.

Reply:
Discussed offline, per https://docs.google.com/document/d/1NzFf9Fi3MLJFG4sZgeVhbxMlxbKumF7xPQ1U3mgquJs we are not doing 0 to disable.

Reply:
Thanks. On further thought, I believe we should continue with 0-to-disable 

I've updated the code here

I've also posted a PR here: https://github.com/googleapis/google-cloud-go/pull/10226